http-server/src/test/java/org/triplea/server/http/spark/SparkServerSystemTest.java

```diff
@@ -1,51 +1,89 @@
 package org.triplea.server.http.spark;
 
-import java.io.BufferedReader;
-import java.io.InputStreamReader;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
+import static com.github.npathai.hamcrestopt.OptionalMatchers.isPresent;
+import static com.github.npathai.hamcrestopt.OptionalMatchers.isPresentAndIs;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
 
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.core.Is;
+import java.net.URI;
 
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.http.client.SendResult;
+import org.triplea.http.client.ServiceClient;
+import org.triplea.http.client.ServiceResponse;
+import org.triplea.http.client.error.report.ErrorReportClientFactory;
+import org.triplea.http.client.error.report.create.ErrorReport;
+import org.triplea.http.client.error.report.create.ErrorReportDetails;
+import org.triplea.http.client.error.report.create.ErrorReportResponse;
+import org.triplea.server.ServerConfiguration;
+import org.triplea.server.reporting.error.upload.ErrorUploadStrategy;
 import org.triplea.test.common.Integration;
 
 import spark.Spark;
 
 
 @Integration
+@ExtendWith(MockitoExtension.class)
```

Comment:
Correct me if I'm wrong, but I'm pretty sure this isn't necessary, because no mockito annotations are being used.

Reply:
d4ebcba48f236a442d754b3ff0971c59f208aa40

You're right, it's only needed when using the annotations.  :+1: